core/domain/blog_domain.py

```diff
@@ -74,18 +125,18 @@ def require_valid_thumbnail_filename(cls, thumbnail_filename, strict=False):
             ValidationError. Provided thumbnail filename is invalid.
         """
         if strict:
-            if not isinstance(thumbnail_filename, str):
-                raise utils.ValidationError(
+            assert isinstance(thumbnail_filename, str), (
```

Comment:
Do not use assert for raising the error here.

Reply:
ok, I will change it back to isinstance. I did it because coverage wont fail if I use assert.

Reply:
One more thing,
Do we need strict and isinstance checks here?
Because mypy is introduced here I don't think we need it. (Introducing new test to fix coverage may need ignores due to mypy).

Reply:
Done